suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -108,6 +108,18 @@ tests:
       polarion-id: CEPH-83573296
 
   - test:
+      config:
```

Comment:
Since we are going with all default values itself for this test case, we can remove all the configs from test suite.

Reply:
Done.